table-project/src/app/page.tsx

```diff
@@ -0,0 +1,10 @@
+import UsersTable from "~/components/UsersTable";
+
+export default function Home() {
+  return (
+    <main>
+      {/* @ts-expect-error Server Component */}
+      <UsersTable/>
+    </main>
+  )
+}
```

Comment:
Ignore error (https://beta.nextjs.org/docs/data-fetching/fetching#asyncawait-in-server-components)